src/main/java/com/solr/clientwrapper/domain/dto/solr/SolrCreateCoreDTO.java

```diff
@@ -0,0 +1,18 @@
+package com.solr.clientwrapper.domain.dto.solr;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@EqualsAndHashCode
+public class SolrCreateCoreDTO {
```

Comment:
Create 1 Request DTO for all Core API's

Reply:
Resolved. Kept only 2 DTOs - SolrSingleCoreDTO & SolrDoubleCoreDTO